packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -27,7 +31,19 @@ export enum ColumnAlign {
   Center = 'center',
 }
 
-export type ColumnDataAccessor<T extends DomainObject> = (rowData: T) => string;
+export type ColumnDataAccessor<T extends DomainObject> = (
+  rowData: T
+) => unknown;
```

Comment:
woah these types for the accessor and formatter are baaad.

Maybe it's essentially for the data type of the underlying row datas .. data. So a number for `invoice.invoiceNumber` or `invoice.pricing.total` etc.. I was thinking to tighten them up we could add another generic to the column so it would be like, `Column<Invoice, number>` for a number column..